humanlayer-wui/src/utils/sessionStatus.tsx

```diff
@@ -6,7 +6,7 @@ export const renderSessionStatus = (session: { status: string; archived?: boolea
 
   // If session is completed but not archived, show "waiting_for_input"
   if (session.status === 'completed' && !session.archived) {
-    return 'waiting_for_input'
```

Comment:
@dexhorthy @allisoneer - Sneaking this in, let me know if you don't like but found the wording to be a little more clear as I was using it.